### PR TITLE
Support symfony 7

### PR DESCRIPTION
**Pull request by Wiktor6**

### CODE REVIEW

**Review by bmulawa — APPROVED** [review]

File: src/Response/Struct/ShipmentAddress.php [hunk 1]
```diff
@@ -47,7 +47,6 @@ protected function setBaseShipmentProperties(array $addressData): void
         $this->streetId = $addressData['streetId'] ?? null;
         $this->streetType = $addressData['streetType'] ?? null;
         $this->streetName = $addressData['streetName'] ?? null;
-        $this->streetType = $addressData['streetType'] ?? null;
```
Comment (olekans) [inline src/Response/Struct/ShipmentAddress.php:50]: why was this removed?

Comment (Wiktor6) [reply src/Response/Struct/ShipmentAddress.php:50]: duplicated, 2 lines above is the the same seter

**Review by olekans — APPROVED** [review]
